linera-service/src/cli/main.rs

```diff
@@ -1723,6 +1723,10 @@ struct ClientOptions {
     #[arg(long)]
     wasm_runtime: Option<WasmRuntime>,
 
+    /// Output log messages from contract execution.
+    #[arg(long)]
+    contract_logs: bool,
```

Comment:
I wonder if this should be `true` for clients by default (and `false` for validators, of course).